gateway-js/src/typings/graphql.ts

```diff
@@ -1,43 +1,11 @@
-import { GraphQLResolveInfo } from 'graphql';
```

Comment:
These types are not used in `gateway-js` they are already declared and now used in `subgraph-js`